src/Bonsai.ML.Torch/ObserveWithGradientTracking.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Reactive;
+using System.Reactive.Linq;
+using TorchSharp;
+using Bonsai;
+using System.ComponentModel;
+
+/// <summary>
+/// Represents an operator that ensures all tensor operations within the observable sequence are executed with gradient tracking enabled.
+/// </summary>
+[Combinator]
+[Description("Ensures all tensor operations within the observable sequence are executed with gradient tracking enabled.")]
+[WorkflowElementCategory(ElementCategory.Combinator)]
+public class ObserveWithGradientTracking
+{
+    /// <summary>
+    /// Returns an observable sequence which is identical to the source sequence, but where all tensor operations are executed with gradient tracking enabled.
+    /// </summary>
+    public IObservable<T> Process<T>(IObservable<T> source)
```

Comment:
I also noticed some of the function signatures seem to have doc strings with missing elements for type parameters, arguments and return values.